docs/markdown/ios/toast.md

```diff
@@ -0,0 +1,158 @@
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/36/0a/c3/360ac34d4f9b31c22058a056abd89e21.jpg" alt="example of a dismissible toast"/>
+    <Do title="Do" />
+    Include a way to dismiss the toast when it is actionable or contains multiple lines of text. Mobile toasts can be dismissed via swiping on the toast in any direction.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/c8/ea/51/c8ea51c06c88110bde0f2854e9103980.jpg" alt="example of leaving a toast on screen"/>
+    <Dont title="Don't" />
+    Leave toasts on screen for a long time without a way to dismiss. Exceptions are blocking error toasts where a user can’t take any action until the error is resolved.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple’s accessibility features, such as reduced transparency, VoiceOver, and increased text size to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple’s Human Interface Guidelines and documentation about accessibility for iOS:
+
+[Accessbile design on Android](https://material.io/design/usability/accessibility.html#understanding-accessibility)
+[Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility)
+
+### Duration
+Some people may take longer to read toasts than others due to aging or other cognitive impairments Use the guide below to set duration for Toasts:
+
+- Brief text of approximately 10–15 words (including button text): 5s
+- Longer than 15 words: Slow readers can read about 125–200 words per minute. Base your duration on the slowest number. For example, a toast with 20 words should be set to 10s. [Source](https://capitalizemytitle.com/reading-time/3000-words/)
+
+## Variants
+
+### Type
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/d9/eb/56/d9eb56243a337584cebbb57e293b3d7f.jpg" noPadding alt="default toast"/>
+    
+    **Default**
+    A generic acknowledgment after an action is taken.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/7f/c3/0d/7fc30d26a72b1c23f71f685f5bfbfc51.jpg" noPadding alt="error toast"/>
+   
+    **Error**
+    Used rarely for connection issues or unknown errors where we don’t want to completely block the users flow, but want the message to persist if the user goes to another surface. Providing a way to solve the error or get help is recommended.
+  </Group>
+  
+</TwoCol>
+
+### Images & graphics
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/02/40/85/024085af164541554f69c18368d7078e.jpg" noPadding alt="text only toast"/>
+    
+    **Text only**
+    A simple, generic acknowledgment after an action is taken These should not be actionable.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/02/40/85/024085af164541554f69c18368d7078e.jpg" noPadding alt="toast with image"/>
```

Comment:
I think this is the wrong image, it has an image :P

Reply:
Fixed, thanks!